qemu/tests/multi_vms_with_stress.py

```diff
@@ -95,6 +96,13 @@ def wait_for_shutdown_all_vms(vms, sessions):
     run_stress_background()
     is_stress_alive()
 
+    if params.get('set_maxcpus'):
+        online_cpu = cpu.online_count() * 2 // 5
```

Comment:
why *2 // 5

the 5 come from cfg is better (example: params.get(num_vms ))

Reply:
thanks a lot, it solved.